lib/Bio/EnsEMBL/DataCheck/Checks/MetaKeyConditional.pm

```diff
@@ -258,4 +259,30 @@ sub has_sift {
   }
 }
 
+sub annotation_source{
+  my ($self) = @_;
+
+  my $mca = $self->dba->get_adaptor('MetaContainer');
+
+ SKIP: {
+     my $method = $mca->single_value_by_key('genebuild.method');
+     if($method ne 'braker' && $method ne 'import') {
+	 skip "Annotation source key not needed for Ensembl builds", 1;
```

Comment:
Can we check with @gnaamati ?
Plants have "external_annotation_import" and "curated" methods.
Should any of them be included in this datacheck?

Reply:
@gnaamati should I resolve this then?

Reply:
@marcoooo this is addressed in the commit
https://github.com/Ensembl/ensembl-datacheck/pull/512/commits/d82b992664b28350123eb1e618db95084ee99fff.
I've resolved the thread. Thanks